2_Functions_Part_2.ipynb

```diff
@@ -77,9 +77,20 @@
   },
```

Comment:
<p>GREAT JOB! </p><p>only thing is your signature should be </p><p><code>def my_format(string, **kwargs): </code></p><p>when getting key-word arguments in a function (the arguments with the "**" ) they are always called **kwaargs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/RB-Data-Analysis-Python-Course/pfc-week05-sagin132/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='85'/>

Reply:
<p>OK</p>